src/components/TopBar/TopBar.tsx

```diff
@@ -310,37 +313,67 @@ export class TopBar extends React.Component<{
                 }}
               />
             </div>
+          </a>
+          {this.props.roomTitle || this.props.roomDescription ? (
```

Comment:
should we all caps this?

Reply:
You mean automatically convert the title to upper case? I think we could just let the user choose?

Reply:
Yeah, you're right it definitely looks better in upper case only.